linux-x64-runner/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+# chia-runner/tasks/main.yml
+- name: Make action-runner directory
+  ansible.builtin.file:
+    path: "{{ path_to_runner_directory }}"
+    state: directory
+    mode: '0755'
+
+- name: Download runner tarball
+  get_url:
+    url: https://github.com/actions/runner/releases/download/v{{runner_release_version}}/actions-runner-linux-x64-{{runner_release_version}}.tar.gz
+    dest: "{{ path_to_runner_directory }}"
+    mode: '0755'
+
+- name: Unarchive runner tarball
+  ansible.builtin.unarchive:
+    src: "{{ path_to_runner_directory }}/actions-runner-linux-x64-{{runner_release_version}}.tar.gz"
+    dest: "{{ path_to_runner_directory }}"
+    remote_src: yes
+    list_files: yes
+    creates: "{{ path_to_runner_directory }}/svc.sh"
+
+- name: Configure the runner for GitHub
+  become_user: ubuntu
```

Comment:
There is a user var in Your defaults file so should probably use that here and not assume Ubuntu in the playbook here. Can default it to Ubuntu in defaults though if you want